src/navigation/effects.js

```diff
@@ -267,7 +267,7 @@ export const emojiPreset = {
         pointerEvents="none"
         style={{
           backgroundColor: 'rgb(51, 54, 59)',
-          height: deviceUtils.dimensions.height,
+          height: deviceUtils.dimensions.height + 50,
```

Comment:
the animation of the modal can be faster I think. But it's up to @christianbaroni